### DISCUSSION TITLE
LovelaceUI layout for the BHyve integration

### DISCUSSION THREAD

**Discussion by jjdonn** [sebr/bhyve-home-assistant#64] [Q&A]
In an earlier thread, SEBR clarified that "You must build your desired layout using the entities which this integration makes available."  I have successfully installed the integration and all BHyve entities are showing in config.  Can anyone suggest options for BHyve LovelaceUI layout that they have used?

Thanks in advance for any suggestions.

Jack

**Comment by disforw** [comment]
Not sure if this is still needed but here is mine.
![311F8419-45E7-469E-B59B-F6B92EB410FE](https://user-images.githubusercontent.com/24241104/180097678-deb6cd5d-5527-4c11-8fc6-103618b11ca1.jpeg)

**Reply by hammermcg** [reply]
Hi @disforw looks good are you able to share some of the code for that lovelace card.  Thanks,

**Reply by disforw** [reply]
```
type: custom:auto-entities
card:
  header:
    type: picture
    image: /local/images/bhyve2.jpg
  type: entities
filter:
  include:
    - entity_id: switch.bhyve_controller_rain_delay
    - entity_id: switch.bhyve_controller*program
    - entity_id: sensor.bhyve_controller_state
    - type: buttons
      entities:
        - entity: switch.bhyve_front_east_side_zone
        - entity: switch.bhyve_front_west_side_zone
        - entity: switch.bhyve_front_sidewalk_zone
        - entity: switch.bhyve_backyard_zone
    - entity_id: sensor.bhyve_hose_state
    - type: buttons
      entities:
        - entity: switch.bhyve_hose_left_zone
          name: Hose Left
        - entity: switch.bhyve_hose_right_zone
          name: Hose Right
  state_color: true
  exclude: []
`

**Reply by hammermcg** [reply]
Thanks @disforw much appreciated
